src/wallet/typings/Account.d.ts

```diff
@@ -0,0 +1,30 @@
+import { ScryptParams } from './core';
+import { WalletAccount } from './Wallet';
+
+/**
+ * This allows for simple utilisation and manipulating of keys without need the long access methods.
+ * Key formats are derived from each other lazily and stored for future access.
+ * If the previous key (one level higher) is not found, it will attempt to generate it or throw an Error if insufficient information was provided (eg. trying to generate private key when only address was given.)
+ * NEP2 <=> WIF <=> Private => Public => ScriptHash <=> Address
+ */
+export class Account {
+  constructor(str: string|object)
+
+  WIF: string
```

Comment:
I believe there is an `encrypted` property now.

Reply:
added